docs/guides/administrator/adding-and-managing-tenants.md

```diff
@@ -0,0 +1,144 @@
+# Adding and Managing Tenants Guide
+
+This guide will show you how to:
+
+* Add tenants during the initial create operation
+* Add new tenant to a running cluster (with public and private keys)
+* Delete a tenant
+
+## Introduction
+
+Opstrace instances support multiple, secured tenants to logically isolate concerns on otherwise shared infrastructure.
+If you are not familiar with what a "tenant" is, see our [tenant key concepts](../../references/concepts.md#tenants) documentation.
+
+Tenants are presented neatly in the Cluster Admin section of our UI:
+
+![tenant overview page](../../assets/tenants-guide-overview-1.png)
+
+Let's walk through an example of adding a new tenant named `newtenant` to a running Opstrace instance named `showdown`.
+
+If you’re coming from the [quick start](../../quickstart.md), and haven’t yet sent data to one of your tenants, stay tuned for our forthcoming integrations guide to make that process easy.
```

Comment:
> stay tuned for our forthcoming integrations guide to make that process easy.

confuses me as a reader... does this refer to content below? does this refer to content being published in the future? how will I find this content? and why am I really interested in reading it?

Reply:
This was intended as a placeholder for linking to future content.  Instead, I will add a TODO in a comment.